src/domain/hanke/edit/components/InfoBulletin.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box } from '@chakra-ui/react';
+import { IconInfoCircle } from 'hds-react';
+
+/** Combine info icon with a text. */
+const InfoBulletin: React.FC<{ info: string }> = ({ info }) => {
+  return (
+    <Box display="flex" alignItems="center" width="max-content" mb="var(--spacing-m)">
+      <IconInfoCircle size="s" />
```

Comment:
According to Figma this should be `IconAlertCircle` instead of `IconInfoCircle`.